src/render/processor.rs

```diff
@@ -22,17 +22,13 @@ pub struct RenderScope {
     pub sample_rate: f32,
 
     pub(crate) node_id: Cell<AudioNodeId>,
-    pub(crate) event_sender: Option<Sender<TriggerEventMessage>>,
+    pub(crate) event_sender: Option<Sender<Event>>,
 }
 
 impl RenderScope {
-    pub(crate) fn send_event(&self, event_type: EventType) {
+    pub(crate) fn send_ended_event(&self) {
```

Comment:
Feels a bit less generic to me than the previous version, but as it is the only event nodes can currently send this is probably very ok

Reply:
Yes you are right, we are losing a bit of flexibility here, but it simplifies the code.
Whenever new types of events are added to the spec we can change it back again